release.py

```diff
@@ -156,10 +156,23 @@ def _create_gh_release(payload, token):
 def _upload_assets(upload_url, token):
     assert token is not None, 'Need token for this action'
     import requests
-    for wheelpath in ['build', 'dist', 'wheelhouse']:
-        wheelfile = os.path.basename(wheelpath)
-        print(f'Uploading wheel {wheelpath}')
-        with open(wheelpath, 'rb') as f:
+    if os.path.exists('wheelhouse'):
+        wheelpaths = [os.path.join('wheelhouse', ff) for ff in os.listdir('wheelhouse')]
+    if wheelpaths is not None:
+        for wheelpath in wheelpaths:
+            wheelfile = os.path.basename(wheelpath)
+            print(f'Uploading wheel {wheelpath}')
+            with open(wheelpath, 'rb') as f:
+                upload_response = requests.post(
+                    f"{upload_url}?name={wheelfile}",
+                    headers={"Authorization": "token " + token,
+                             "Content-type": "application/octet-stream"},
+                    data=f.read())
+                print(upload_response.text)
+    mltbx = os.path.join('mltbx', 'spinw.mltbx')
+    if os.path.exist(mltbx):
```

Comment:
Similar question here - what happens if it doesn't exist!

Reply:
In both cases if the wheels or `mltbx` file doesn't exist, it doesn't get uploaded (to PyPI for wheels or the github release for the `mltbx` but the CI doesn't fail... you'd just have to manually upload them...

Reply:
OK, I'm just wondering how we'd know if the upload failed though and we'd have to do it manually?

Reply:
Ok... I've added code to error if there are no wheels or mltbx files to upload.